working/augmentation-libraries/feature-specification.md

```diff
@@ -474,7 +487,7 @@ The augmenting function does not have to pass the same arguments to
 `augmented()` as were passed to it. It may call it once, more than once, or not
 at all.
 
-It is a compile-time error if:
+It is a compile-time error if, after type inheritance:
```

Comment:
(Type inference can't change the type of the augmenting declaration, it inherits its types, but it can only do so after type inference has given types to its augmented declaration, which is why this is "after type inference".)